shared/teams/emojis/add-emoji.tsx

```diff
@@ -577,10 +580,12 @@ const styles = Styles.styleSheetCreate(() => ({
   }),
   footerContainer: Styles.platformStyles({
     isElectron: {
-      paddingBottom: Styles.globalMargins.xsmall,
-      paddingLeft: Styles.globalMargins.small,
-      paddingRight: Styles.globalMargins.small,
-      paddingTop: Styles.globalMargins.xsmall,
+      ...Styles.padding(
+        Styles.globalMargins.xsmall,
+        Styles.globalMargins.small,
```

Comment:
i belive you can skip the last 2 if its like this

Reply:
magic